cmd/api/src/go.mod

```diff
@@ -47,6 +47,7 @@ require (
 	github.com/unrolled/secure v1.13.0
 	github.com/zenazn/goji v1.0.1
 	go.uber.org/mock v0.2.0
+	go.uber.org/zap v1.13.0
```

Comment:
It's curious how this new dependency got added. It doesn't look like it's used in any of the files in this PR.

Reply:
Let's `zap` this dependency if we aren't using it. Pun intended. Suffer.

Reply:
![image](https://media1.tenor.com/m/iGhUqblPyqAAAAAC/xx.gif)